app/services/participant_outcomes/create.rb

```diff
@@ -35,7 +35,9 @@ def create_outcome
         @created_outcome = if outcome_already_exists?
                              latest_existing_outcome
                            else
-                             build_outcome.tap(&:save!)
+                             outcome = build_outcome.tap(&:save!)
+                             outcome.application.update!(participant_outcome_state: state)
```

Comment:
instead of doing this here, I was thinking we add a feature flag where we read from `participant_outcome_state` and instead read it from the declaration outcomes instead? this way we will avoid having to backfill any data, happy to discuss this one